packages/@react-aria/focus/test/focusSafely.test.js

```diff
@@ -14,14 +14,20 @@
 import {act, render} from '@react-spectrum/test-utils';
 import {focusSafely} from '../';
 import React from 'react';
-import * as ReactAriaUtils from '../../utils/index';
+import * as ReactAriaUtils from '@react-aria/utils';
 import {setInteractionModality} from '@react-aria/interactions';
 
+jest.mock('@react-aria/utils', () => {
+  let original = jest.requireActual('@react-aria/utils');
+  return {
+    ...original,
+    focusWithoutScrolling: jest.fn()
+  };
+});
```

Comment:
You can't spy on a module export with swc, which errors with "cannot redefine property". This uses jests module mocks to do that instead.